test/base/SpecBase.scala

```diff
@@ -67,6 +67,7 @@ trait SpecBase
   val incidentIndex: Index              = Index(0)
   val sealIndex: Index                  = Index(0)
   val houseConsignmentIndex: Index      = Index(0)
+  val departureIndex: Index             = Index(0)
```

Comment:
Do we really need to add this in seeing as we already have dtmIndex and others that can be used?